pkg/action/action.go

```diff
@@ -220,6 +233,11 @@ func (cfg *Configuration) renderResources(ch *chart.Chart, values chartutil.Valu
 		if err != nil {
 			return hs, b, notes, errors.Wrap(err, "error while running post render on files")
 		}
+
+		err = postRenderHookManifests(hs, pr)
+		if err != nil {
+			return hs, b, notes, err
+		}
```

Comment:
Two things come to mind with reviewing this...
1. The Helm project follows strict semantic versioning. If someone is using a post renderer today and has hooks there is no effect. If they use it in a version with this pull request merged their content will be changed. Where we [document the backwards compatibility](https://github.com/helm/community/blob/main/hips/hip-0004.md) rules we note that someone using Helm and a chart before and after an upgrade should see no difference. They could with the current code. I think this change needs to be behind an opt-in flag on the CLI and opt-in property on the Go API.
2. This changes the behavior of post rendering. Instead of being run once per chart with everything in one available to the script the post renderer is run on each hook individually. I wonder what the implications of that are.